lms/envs/common.py

```diff
@@ -1495,6 +1495,18 @@
         ],
         'output_filename': 'css/certificates-style-rtl.css'
     },
+    'style-leaner-dashboard': {
```

Comment:
While we are aiming for a leaner learner dashboard, I definitely meant to type ```learner``` :)